### DISCUSSION TITLE
Screen is not visible from mobile when share from desktop

### DISCUSSION THREAD

**Discussion by basit-skill-lync** [screego/server#149] [Q&A]
Screen is coming blank when opened room from mobile while sharing it from desktop. Can you tell why is it happening

**Comment by jmattheis** [comment]
See https://screego.net/#/faq?id=stream-doesnt-load

**Reply by basit-skill-lync** [reply]
Hey @jmattheis , just need to understand will screen be visible from mobile if shared from desktop/laptop?
Am I missing something in the configuration or is it developed only for desktop/laptop.

**Reply by jmattheis** [reply]
In theory, yes. But there are some problems with iPhones, but on Android it should work.
